app/views/public_pages/stimulus.html.erb

```diff
@@ -41,153 +66,34 @@
-            <p><%=t("views.public_pages.stimulus_payment.faq.eip.when.a") %></p>
-          </div>
-        </div>
-        <div class="accordion">
-          <a href="#" class="accordion__button" aria-expanded="true" aria-controls="a11">
-            <h3><%=t("views.public_pages.stimulus_payment.faq.eip.other_benefits.q") %></h3>
-          </a>
-          <div class="accordion__content" id="a11">
-            <p><%=t("views.public_pages.stimulus_payment.faq.eip.other_benefits.a") %></p>
-          </div>
-        </div>
-        <div class="accordion">
-          <a href="#" class="accordion__button" aria-expanded="true" aria-controls="a12">
-            <h3><%=t("views.public_pages.stimulus_payment.faq.eip.deadline.q") %></h3>
-          </a>
-          <div class="accordion__content" id="a12">
-            <p><%=t("views.public_pages.stimulus_payment.faq.eip.deadline.a") %></p>
-          </div>
-        </div>
-        <div class="accordion">
-          <a href="#" class="accordion__button" aria-expanded="true" aria-controls="a13">
-            <h3><%=t("views.public_pages.stimulus_payment.faq.eip.owe_taxes.q") %></h3>
-          </a>
-          <div class="accordion__content" id="a13">
-            <p><%=t("views.public_pages.stimulus_payment.faq.eip.owe_taxes.a") %></p>
-          </div>
-        </div>
-        <div class="accordion">
-          <a href="#" class="accordion__button" aria-expanded="true" aria-controls="a14">
-            <h3><%=t("views.public_pages.stimulus_payment.faq.eip.one_time.q") %></h3>
-          </a>
-          <div class="accordion__content" id="a14">
-            <p><%=t("views.public_pages.stimulus_payment.faq.eip.one_time.a") %></p>
-          </div>
-        </div>
-        <div class="accordion">
-          <a href="#" class="accordion__button" aria-expanded="true" aria-controls="a15">
-            <h3><%=t("views.public_pages.stimulus_payment.faq.eip.additional_eip.q") %></h3>
-          </a>
-          <div class="accordion__content" id="a15">
-            <p><%=t("views.public_pages.stimulus_payment.faq.eip.additional_eip.a") %></p>
-          </div>
-        </div>
-        <div class="accordion">
-          <a href="#" class="accordion__button" aria-expanded="true" aria-controls="a16">
-            <h3><%=t("views.public_pages.stimulus_payment.faq.eip.scams.q") %></h3>
-          </a>
-          <div class="accordion__content" id="a16">
-            <p><%=t("views.public_pages.stimulus_payment.faq.eip.scams.a") %></p>
-          </div>
-        </div>
+        <% t("views.public_pages.stimulus_payment.faq.eip.items").each do |faq| %>
+          <% @faq_counter += 1 %>
+          <div class="accordion">
+            <a href="#" class="accordion__button" aria-expanded="true" aria-controls=<%= "a#{@faq_counter}" %>
+                <h3><%= faq[:title] %></h3>
+            </a>
+            <div class="accordion__content" id="a1">
+              <p><%= faq[:content_html].html_safe %></p>
```

Comment:
I'm surprised you need `.html_safe` since I would figure that the `content_html` is already marked as safe. It's fine as-is, but I'm curious if it also works properly without the `.html_safe`. Not a blocker.